kolibri/core/tasks/tasks.py

```diff
@@ -0,0 +1,37 @@
+"""
```

Comment:
The task defined in here should live in `kolibri/core/content/tasks.py`

Reply:
Okay, will do. 

Also, I've some questions regarding separation of tasks (related to breaking changes..). Will talk to you on Slack about those!